tests/functional/karate_tests/src/test/java/logback-test.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does this use log4j2 AND logback? can we use one or the other?

Reply:
Removed log4j2 support